python/src/PointToFieldFunctionalChaosAlgorithm_doc.i.in

```diff
@@ -0,0 +1,274 @@
+%feature("docstring") OT::PointToFieldFunctionalChaosAlgorithm
+"Functional metamodel algorithm based on chaos decomposition.
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+The present algorithm allows one to build a response surface of the application :math:`h` of the form:
+
+.. math::
+    h: \left|
+      \begin{array}{ccl}
+         \Rset^p & \rightarrow & \cM_N \times (\Rset^d)^N \\
+         \mat{X} & \mapsto & \vect{Y}
+      \end{array}
+    \right.
+
+with :math:`\cM_N` a mesh of :math:`\cD \subset \Rset^n`.
```

Comment:
What is $d$ then? Is $n$ an observation of the random variable $N$?

Reply:
it clearly appears in the first expression as the field dimension